login-workflow/example/src/App.tsx

```diff
@@ -12,8 +13,100 @@ import { ExampleHome } from './screens/ExampleHome';
 import { routes } from './navigation/Routing';
 
 import productLogo from './assets/images/eaton_stacked_logo.png';
+// import { Button, CardActions, CardContent, CardHeader, Divider, Typography } from '@material-ui/core';
+// import { Link, Route } from 'react-router-dom';
 // import { ExamplePreAuth } from './screens/ExamplePreAuth';
-// import { Route } from 'react-router-dom';
+// import { EmptyState } from '@pxblue/react-components';
+// import Person from '@material-ui/icons/Person';
+
+// const customRegistrationSuccessScreen = (registrationData: RegistrationData | undefined): JSX.Element => (
```

Comment:
Can we move these to separate files to clean up App.tsx a little bit?

Reply:
You got it!